docs/README.md

```diff
@@ -125,8 +125,9 @@ The component rendered when used the `checkout-summary.compact` block.
 | Prop name | Type | Required |
 | --- | --- | --- |
 | [`totalizers`](#summarysmall-totalizers) | `array` | `true` |
-| [`totalizersToShow`](#summarysmall-totalizerstoshow) | `array` | `true` |
+| [`totalizersToShow`](#summarysmall-totalizersToShow) | `array` | `true` |
```

Comment:
This prop doesn't seem to be required for the component to work

Reply:
Nice catch!